atomium/atomium.api.php

```diff
@@ -57,7 +57,7 @@ function hook_atomium_definition_hook() {
  */
 function hook_atomium_definition_form_hook($form, &$form_state) {
   $form['component'] = array(
-    '#theme' => 'username',
+    '#theme' => array('username'),
```

Comment:
why we do not use _atomium_extend_theme_hook function?

Reply:
Because it's an example, do you think it worth it ?